src/app/CommandHandler.h

```diff
@@ -233,6 +233,11 @@ class CommandHandler
     friend class TestCommandInteraction;
     friend class CommandHandler::Handle;
 
+    // ExchangeDelegate
+    CHIP_ERROR OnMessageReceived(Messaging::ExchangeContext * apExchangeContext, const PayloadHeader & aPayloadHeader,
+                                 System::PacketBufferHandle && aPayload) override;
+    void OnResponseTimeout(Messaging::ExchangeContext * apExchangeContext) override {}
```

Comment:
Shouldn't this have a `VerifyOrDie` here because we wouldn't be expecting a response to an InvokeResponse right?

Reply:
agree, fixed, thanks